src/Kriti/Parser/Grammar.y

```diff
@@ -0,0 +1,220 @@
+{
+module Kriti.Parser.Grammar where
+
+import qualified Data.Aeson as J
+import Data.Bifunctor (first)
+import qualified Data.HashMap.Strict as M
+import qualified Data.List as List
+import qualified Data.Scientific as S
+import qualified Data.Text as T
+import qualified Data.Vector as V
+import GHC.Generics
+import qualified Kriti.Error as E
+import qualified Kriti.Parser.Lexer as L
+import Kriti.Parser.Monad
+import Kriti.Parser.Token
+import Kriti.Parser.Spans
+
+}
+
+%name parser term
```

Comment:
Here is our parser specification. It should be fairly easy to grok, but let me know if you have any specific questions.